### ISSUE TITLE
Odd collisions

### ISSUE THREAD

**Issue by Heebe** [torch2424/Fight#136]
You can run around the map and at certain areas, if a boss is actively chasing you, he will be frozen in that spot. So far only found 2 spots.

This spot at the **bottom** of the map:
![untitled](https://cloud.githubusercontent.com/assets/7026914/9422788/a7040508-4859-11e5-95a5-5823fd7f17fb.png)
This spot at the **top** of the map:
![untitled2](https://cloud.githubusercontent.com/assets/7026914/9422789/a7045670-4859-11e5-88ea-4a204088e3a4.png)

**Comment by Heebe** [comment]
Also was able to push the boss into the side of the map as well:
![untitled3](https://cloud.githubusercontent.com/assets/7026914/9422806/1c038b80-485a-11e5-8e9d-b1b860ab1994.png)

**Comment by torch2424** [comment]
Yeah I think it has to relate to the #123 , But I noticed this as well, I kinda wished it would go away haha, I'll add this to the backburner, probaly be fixed in the first beta
